lib/os/Kconfig

```diff
@@ -157,6 +157,14 @@ config UTF8
 	  Enable the utf8 API. The API implements functions to specifically
 	  handle UTF-8 encoded strings.
 
+config MEM32
```

Comment:
```suggestion
config MEMCPY32
```

Reply:
The idea was that compiling a C function is cheap, but invoking gcc on a source file is expensive.

So I want to gather related, future, functions in this mem32 source file.

Different alignment requirements, granularity, memset, etc.